app/routes/dev.tsx

```diff
@@ -56,6 +57,30 @@ export const action = async ({ request }: ActionArgs) => {
         result: 'ok',
         message: 'DNS record creation requested',
       });
+    case 'update-dns-record-request':
+      await updateDnsRequest({
+        id: 62,
+        type: 'A',
+        name: 'osd700-a2.user1.starchart.com',
+        value: '192.168.0.2',
+        username: 'user1',
+      });
+      return json({
+        result: 'ok',
+        message: 'DNS record creation requested',
+      });
+    case 'delete-dns-record-request':
+      await deleteDnsRequest({
```

Comment:
Is there any necessity of passing so many arguments to `deleteDnsRequest`?

Since this is deletion maybe the function could just take an id and just make a deletion by id?

Reply:
`id` is to remove a record in DB. But I need type, name, value to remove in Route53. It doesn't use unique identifier.

Reply:
It's an interesting point though: could you not do a query to the db by the `id` to get the rest?  Not critical to change it all, just thinking out loud.

Reply:
It could remove without `id` and with type, name, value, it should be unique combination. But I'm not sure if we should do in a less reliable way.

Reply:
I think Prisma does not allow me to use delete without unique key. 

`Type '{ type: RecordType; name: string; value: string; }' is not assignable to type 'RecordWhereUniqueInput'.
  Object literal may only specify known properties, and 'type' does not exist in type 'RecordWhereUniqueInput'.ts(2322)`